app/src/androidTest/java/com/github/palFinderTeam/palfinder/UIMockMeetUpRepositoryModule.kt

```diff
@@ -91,13 +93,53 @@ object UIMockMeetUpRepositoryModule {
         ): Flow<Response<List<MeetUp>>> {
             return flow {
                 val meetUps = db.values.filter { meetUp ->
-                    meetUp.location.distanceInKm(location)*1000 <= radiusInM
+                    meetUp.location.distanceInKm(location) * 1000 <= radiusInM
                 }
 
                 emit(Response.Success(meetUps))
             }
         }
 
+        override suspend fun joinMeetUp(
+            meetUpId: String,
+            userId: String,
+            now: Calendar
+        ): Response<Unit> {
+            return if (db.containsKey(meetUpId)) {
+                val meetUp = db[meetUpId] ?: return Response.Failure("Could not find meetup")
```

Comment:
Would be good to have the faillure message be defined inside constant so that in case we want to display them to the user we can pattern match on them easily.